### ISSUE TITLE
 Feature Request: Display Lemmy's images

### ISSUE THREAD

**Issue by AndreFurtadoCoelho** [Ranchero-Software/NetNewsWire#4048]
Hey, first post ever here so don't bully me if the structure is not great.

I'd like to request a feature that was working flawlessly on subreddits and unfortunatelly not working now that I've moved to Lemmy.

I've successfully added a few lemmy communities to NetNewsWire and the app is working great fetching all the posts, however I'm not having quite the same experience I was having with subreddits.
Whatever post that I'd be reading in NetNewsWire from reddit would automatically load the pictures from that post in the app. The same behavior is not happening on Lemmy's communities.

I'm not sure I'll be the only one wanting this feature but this is probaly THE FEATURE that made fall in love with NetNewsWire since it was a breeze scrolling through r/watches. Now I have basically to open every single post directly on the website.

Btw, I've never had the chance to thank you for your great work.
One of the best apps I use.

PS: fuck u/spez

**Comment by markcellus** [comment]
Yeah do the feeds from Open RSS work? I think they have the images I believe. At least until NetNewsWire implements this feature.

https://openrss.org/lemmy.ml/c/linux
